src/trees/projects-tree.ts

```diff
@@ -49,11 +57,42 @@ export class ProjectsTree
     projectOrIssue?: RedmineProject | Issue
   ): Promise<(RedmineProject | Issue)[]> {
     if (projectOrIssue != null && projectOrIssue instanceof RedmineProject) {
+      if (this.viewStyle == ProjectsViewStyle.TREE) {
+        const subprojects = this.projects.filter((project) => {
+          if (project.parent) {
+            return project.parent.id == projectOrIssue.id;
+          } else {
+            return false;
+          }
```

Comment:
I would like to understand this line. It is just a bit confusing for me because I generally use C language. I see that parent option can be undefined if it is not a subproject. So when first expression is false, will second expression be executed or just skipped ?

Reply:
It will be skipped. JavaScript evaluates such expressions lazily, so in scenario `expr1 && expr2`, if `expr1` is falsy, then `expr2` won't be executed at all. Similar to `expr1 || expr2`, if `expr1` will be truthy, then `expr2` won't be executed at all.

Reply:
I really liked it. Thank you 😄😄 
Usefull information, It will come in handy when solving other issues 🤓